module/VuFind/src/VuFind/Connection/Wikipedia.php

```diff
@@ -418,7 +418,10 @@ protected function parseWikipedia($rawBody)
         $imageName = null;
         $imageCaption = null;
         // Check if data exists or not
-        if (isset($rawBody['query']['pages']['-1'])) {
+        if (isset($rawBody['query'])
```

Comment:
Why is it necessary to do three levels of checking here? I thought isset was safe to call on deeply nested structures; if it is not, I'm sure there are many other places in the code where revisions would be necessary... but that seems to work against readability!

Reply:
You are right, I overlooked bad types in doc comments, should be fixed now